crates/runtime/src/accelerated_table/refresh.rs

```diff
@@ -362,9 +372,18 @@ impl Refresher {
         }
     }
 
+    async fn refresh_append_overlap(&self) -> u128 {
```

Comment:
It would be clearer that this returns nanoseconds if it was either added as a comment to the function, or the function name changed to `refresh_append_overlap_nanos`